OWNERS_ALIASES

```diff
@@ -0,0 +1,30 @@
+aliases:
+  tech-leads:
+    - csrwng
+    - enxebre
+    - sjenning
+  core-approvers:
+    - bryan-cox
+    - csrwng
+    - enxebre
+    - jparrill
+    - muraee
+    - sjenning
+  core-reviewers:
+    - bryan-cox
+    - csrwng
+    - devguyio
+    - enxebre
+    - jparrill
+    - muraee
+    - sdminonne
+    - sjenning
+  kubevirt-approvers:
+    - davidvossel
+    - nirarg
+  kubevirt-reviewers:
+    - davidvossel
+    - nirarg
+  ibm-reviewers:
```

Comment:
We might want to check to see if they want anyone else added.

Reply:
We do but can add them in a follow-up PR.